recipes/lksctp-tools/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, chdir, copy, rm, rmdir, collect_libs
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class LKSCTPToolsConan(ConanFile):
+    name = "lksctp-tools"
+    settings = "os", "arch", "compiler", "build_type"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/sctp/lksctp-tools"
+    description = "The Linux Kernel Stream Control Transmission Protocol (lksctp) project"
+    package_type = "library"
+    topics = ("sctp")
+    license = "GPL-2.0, LGPL-2.1"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.cppstd")
+        self.settings.rm_safe("compiler.libcxx")
+
+    def validate(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration(f"{self.ref} is only available on Linux")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = AutotoolsToolchain(self)
+        yes_no = lambda v: "yes" if v else "no"
+        tc.configure_args.extend([
+            "--disable-tests",
+            "--enable-static={}".format(yes_no(not self.options.shared)),
```

Comment:
```suggestion
        tc.configure_args.extend([
            "--disable-tests",
```

--enable-static, --enable-shared, --disable-static and --disable-shared are not needed as you are using AutotoolsToolchain: https://docs.conan.io/2/reference/tools/gnu/autotoolstoolchain.html#customization